ibft/ibft.go

```diff
@@ -85,6 +87,8 @@ func New(
 }
 
 func (i *ibftImpl) Init() {
+	i.listenToDecidedQueueMessages()
+	i.listenToSyncQueueMessages(i.ValidatorShare.PublicKey.Serialize()) // pass the pubkey byte to avoid deadlock
```

Comment:
@nivBlox we have the identifier set in the sturct... why pass another similar argument here?

Reply:
cause sync is indexing only with pubkey (role is not relevant for this type of msg's)